E2_ESRGAN/lib/utils.py

```diff
@@ -1,17 +1,24 @@
 import tensorflow as tf
 from lib.settings import settings
 
-def checkpoint(checkpoint, load=False):
+
+def checkpoint(checkpoint, training_phase, load=False, assert_consumed=True):
   """ Saves or Loads checkpoint.
```

Comment:
Split this into two methods. Saving & loading are two very separate behaviors and it seems weird in terms of modularity to group them together.

Reply:
SG. :P Fixed it :)